lib/sisimai/bite/email/ezweb.rb

```diff
@@ -0,0 +1,260 @@
+                v = dscontents[-1]
+              end
+
+              r = Sisimai::Address.s3s4(cv[1])
+              if Sisimai::RFC5322.is_emailaddress(r)
+                v['recipient'] = r
+                recipients += 1
+              end
+
+            elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+              # Status: 5.1.1
+              # Status:5.2.0
+              # Status: 5.1.0 (permanent failure)
+              v['status'] = cv[1]
+
+            elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+              # Action: failed
+              v['action'] = cv[1].downcase
+
+            elsif cv = e.match(/\A[Rr]emote-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+              # Remote-MTA: DNS; mx.example.jp
+              v['rhost'] = cv[1].downcase
+
+            elsif cv = e.match(/\A[Ll]ast-[Aa]ttempt-[Dd]ate:[ ]*(.+)\z/)
+              # Last-Attempt-Date: Fri, 14 Feb 2014 12:30:08 -0500
+              v['date'] = cv[1]
+
+            else
+              next if Sisimai::String.is_8bit(e)
+              if cv = e.match(/\A[ \t]+[>]{3}[ \t]+([A-Z]{4})/)
+                #    >>> RCPT TO:<******@ezweb.ne.jp>
+                v['command'] = cv[1]
+
+              else
+                # Check error message
+                if rxmessages.find { |a| e =~ a }
+                  # Check with regular expressions of each error
+                  v['diagnosis'] ||= ''
+                  v['diagnosis']  += ' ' + e
+
+                else
+                  # >>> 550
+                  v['alterrors'] ||= ''
+                  v['alterrors']  += ' ' + e
+                end
+              end
+            end
+          end
+        end
+        return nil if recipients.zero?
+
+        dscontents.map do |e|
+          if e['alterrors'] && e['alterrors'].size > 0
+            # Copy alternative error message
+            e['diagnosis'] ||= e['alterrors']
+            if e['diagnosis'] =~ /\A[-]+/ || e['diagnosis'] =~ /__\z/
+              # Override the value of diagnostic code message
+              e['diagnosis'] = e['alterrors'] if e['alterrors'].size > 0
+            end
+            e.delete('alterrors')
+          end
+          e['diagnosis'] = Sisimai::String.sweep(e['diagnosis'])
+
+          if mhead['x-spasign'] && mhead['x-spasign'] == 'NG'
+            # Content-Type: text/plain; ..., X-SPASIGN: NG (spamghetti, au by EZweb)
+            # Filtered recipient returns message that include 'X-SPASIGN' header
+            e['reason'] = 'filtered'
+
+          else
+            if e['command'] == 'RCPT'
+              # set "userunknown" when the remote server rejected after RCPT
+              # command.
+              e['reason'] = 'userunknown'
+            else
+              # SMTP command is not RCPT
+              catch :SESSION do
+                ReFailure.each_key do |r|
+                  # Verify each regular expression of session errors
+                  ReFailure[r].each do |rr|
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->